tests/nlp/utils/test_properties.py

```diff
@@ -38,3 +38,17 @@ def test_calculate_default_properties():
 
     # Assert
     assert_that(result, equal_to({'Toxicity': [0] * 20_000}))
+
+
+def test_skipping_long_calculation_properties(tweet_emotion_train_test_textdata):
```

Comment:
Add a test that verifies if those tests can handle different types of None in the text (behaviour should be to ignore them)

Reply:
Done